pkg/pkg.mk

```diff
@@ -0,0 +1,22 @@
+#
+# Include this file if your Package needs to be checked out by git
+#
+PKG_DIR?=$(CURDIR)
+PKG_BUILDDIR?=$(BINDIR)/pkg/$(PKG_NAME)
+
+download: $(PKG_BUILDDIR)/.downloaded
+
+$(PKG_BUILDDIR)/.downloaded:
+	mkdir -p $(PKG_BUILDDIR)
+	$(GITCACHE) clone "$(PKG_URL)" "$(PKG_VERSION)" "$(PKG_BUILDDIR)"
```

Comment:
I called it "pkg.mk" because we will extend it, and then either some logic will decide wether a pkg is "git", or the targets can be called "git-checkout", or whatever.